<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>121</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>121</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>121</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Уведомление</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/icon.png</normaloff>../img/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid white;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="msg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>311</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>500</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Rex Bold </family>
       <pointsize>18</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
border: 0px</string>
     </property>
     <property name="text">
      <string>Поиск фото закончен</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_close">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(255, 255, 255);
	color: black;
	border-radius: 9px;
	height: 22px;
}

QPushButton:hover {
	background-color: rgb(208, 208, 208);
	color: rgb(45, 45, 45);
}

QPushButton:pressed {
	background-color: rgb(207, 207, 207);
}</string>
     </property>
     <property name="text">
      <string>закрыть</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
